packages/browser/src/node/providers/webdriver.ts

```diff
@@ -80,6 +80,14 @@ export class WebdriverBrowserProvider implements BrowserProvider {
       capabilities: this.buildCapabilities(),
     })
 
+    if (this.ctx.config.browser.ui) {
+      await this.browser.maximizeWindow()
+    }
+    else {
+      const { width, height } = this.ctx.config.browser.viewport
+      await this.browser.setWindowSize(width, height)
```

Comment:
You do it here by not in playwright, is this intentional? Do you think we should keep viewport only for the iframe?

Reply:
There is no need (intentional), PW will use the viewport size properly from the options, WDIO will ignore them.

Reply:
> Do you think we should keep viewport only for the iframe?

Yes, why not?